.drone.yml

```diff
@@ -88,3 +88,12 @@ pipeline:
     secrets: [slack_webhook]
     commands:
       - node ./scripts/slack-unpublished.js
+  
+  publish-icons:
+    when:
+      event: push
+      branch: master
+    image: registry.usw.co/cloud/node-12-puppeteer:03228dd2d438979e8973ec16351c4d7bc88f552e
```

Comment:
check this image vs the one in money-includes

Reply:
Updated